content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -333,6 +334,22 @@ The availability of Pod DNS Config and DNS Policy "`None`" is shown as below.
 | 1.10 | Beta (on by default)|
 | 1.9 | Alpha |
 
+## Windows {#dns-windows}
+
+- ClusterFirstWithHostNet is not supported. Windows treats all names with a
```

Comment:
```suggestion
- `ClusterFirstWithHostNet` is not supported on Windows nodes. Windows treats all names
  that contain a
```

Reply:
I was trying to avoid adding 'Windows node' here since `ClusterFirstWithHostNet` is a field specified in pod specs.

How does `ClusterFirstWithHostNet` is not supported for pods running on Windows nodes? 

What is and isn't supported on the nodes does seem to bleed through to what should and should not be specified in the pod specs. I'm not sure if should worry about that though.